samples/cpp/benchmark_app/inputs_filling.cpp

```diff
@@ -250,7 +250,7 @@ ov::Tensor create_tensor_random(const benchmark_app::InputInfo& inputInfo,
     size_t tensor_size =
         std::accumulate(inputInfo.dataShape.begin(), inputInfo.dataShape.end(), 1, std::multiplies<size_t>());
     auto tensor = ov::Tensor(inputInfo.type, inputInfo.dataShape);
-    auto data = tensor.data<T>();
+    auto data = inputInfo.type == ov::element::boolean ? static_cast<T*>(tensor.data()) : tensor.data<T>();
```

Comment:
should we call this template with `bool` type then? currently, it's called with u8 for bool

Reply:
@ilya-lavrenov yeah, bool type fixes the issue, but sizeof(bool) is implementation defined and might differ from 1 (I'm not sure we support such platforms?), and theoretically it may cause problems with pointers arithmetic in buffer filling function. Can we consider this change as safe enough? If so, that would be the best way to fix the issue